test/meson.build

```diff
@@ -64,16 +64,35 @@ fakerand = executable(
   'fakerand.c',
 )
 
-rauc_t = find_program(
-  'rauc.t'
-)
+pytest = find_program('pytest', required: false)
 
-test(
-  'rauc.t',
-  rauc_t,
-  args : '-v',
-  is_parallel : false,
-  timeout : 360,
-  env : ['SHARNESS_BUILD_DIRECTORY=' + meson.build_root()],
-  workdir : meson.current_source_dir(),
-)
+if pytest.found()
+  pytests = [
+    'test_cmdline',
+    'test_rauc_info',
```

Comment:
`test_cmdline` is a generic test for some comment line things while the other test were meant to test the subcommand they are named after. However, this is not fully true since I have tests named `test_rauc_sign` or `test_rauc_verify`, too. Suggestions welcome.

Reply:
I'd just drop the `_rauc` part, as they all apply to RAUC.